app/src/main/java/com/kickstarter/services/ApolloClientType.kt

```diff
@@ -36,6 +36,8 @@ interface ApolloClientType {
 
     fun getProjectComments(slug: String, cursor: String?, limit: Int = 25): Observable<CommentEnvelope>
 
+    fun getRepliesForComment(comment: Comment, cursor: String = "", pageSize: Int = 25): Observable<CommentEnvelope>
```

Comment:
Great job @Arkariang . Just tiny comment; can we rename ``` pageSize ``` to ``` limit ``` or ``` limit ``` to ``` pageSize ``` for consistency?. Otherwise looks good

Reply:
We have const on my brach for page size we can use it after merge

Reply:
I'll tackle that comment on https://kickstarter.atlassian.net/browse/NT-1969